test/androidTest/java/org/thoughtcrime/securesms/espresso/ViewActions.java

```diff
@@ -30,6 +40,31 @@ public void perform(UiController uiController, View view) {
         };
     }
 
+    public static ViewAction clickPercent(final float pctX, final float pctY){
+        return new GeneralClickAction(
+                Tap.LONG,
```

Comment:
remove 1 indent

Reply:
how did this indent happen??
why??
how?? -1/2